resources/assets/coffee/_classes/forum.coffee

```diff
@@ -137,6 +139,17 @@ class @Forum
 
     true
 
+  keyboardNavigation: (e) =>
+    return if $(e.target).is('input') or not @_postsCounter.length
```

Comment:
I think [this](https://github.com/ccampbell/mousetrap/blob/master/mousetrap.js#L986) is the complete list of inputs.